main.cpp

```diff
@@ -107,10 +107,36 @@ struct CarWash
     you should be able to deduce the return type of those functions based on their usage in Person::run()
     You'll need to insert the Person struct from the video in the space below.
  */
+struct Limb
+{
+    int steps;
```

Comment:
initialize your variables!
You're adding `1` to it in `stepForward()`.   what is its final value after that addition?
If you don't initialize your variable, you'll have a garbage value before and after that addition
- [ ] revise